packages/date-picker/theme/material/vaadin-date-picker-overlay-content-styles.js

```diff
@@ -140,16 +140,15 @@ registerStyles(
       border-top: 1px solid var(--material-divider-color);
     }
 
-    [part='cancel-button'] {
+    ::slotted([slot='cancel-button']) {
```

Comment:
Not happy about this selector, targeting individual buttons now requires an even more obscure selector than before.

Also https://github.com/vaadin/platform/issues/3149 mentions:

> No need to deal with obscure selectors like :host, ::slotted, and [part~="..."]

No idea what to do about it though.

Reply:
> No need to deal with obscure selectors like :host, ::slotted, and [part~="..."]

This is what users currently have to deal with because of using `ThemableMixin`.
So the change itself makes it easy for users to just use `vaadin-button`.

We would have to use this selector as we already do elsewhere, for example:

https://github.com/vaadin/web-components/blob/3926c3ada22533bafcf3d65429a0b6a3e78b130b/packages/dialog/src/vaadin-dialog.js#L33-L35

Reply:
Well for custom global styles the selector would be something like:
```
vaadin-date-picker-overlay-content [slot='cancel-button']
```
That seems reasonable. Still a bit confusing that some elements needs to be targeted with `::part` and others with `[slot]`.

Reply:
> So the change itself makes it easy for users to just use vaadin-button.

I meant targeting the today or cancel button individually.

Reply:
I see. In that case, custom styles could look like this:

```css
vaadin-date-picker-overlay-content > vaadin-button[slot="today-button"] {
  /* button styles */
}
```

We could also consider having a single slot, e.g. `toolbar`.

Reply:
Yeah, I was a bit confused from looking at the `::slotted` selector, but as long as devs don't have to use that with the new (global) styling approach it should be fine. Devs using the old approach (styles injected into shadow DOM) will end up with a more cryptic selector though.

> We could also consider having a single slot, e.g. toolbar.

I think **must** be possible to target the buttons individually, so individual slots are definitely better than a single slot. If you want to target both, you can still use the toolbar part.